pandas/tests/test_expressions.py

```diff
@@ -332,78 +331,81 @@ def testit():
         expr.set_numexpr_threads()
         testit()
 
-    def test_bool_ops_raise_on_arithmetic(self):
+    @pytest.mark.parametrize(
+        "op_str,opname", list(zip(["/", "//", "**"], ["truediv", "floordiv", "pow"]))
```

Comment:
Do you find `list(zip(...)` clearer than writing out the pairs?

```python
[('/', 'truediv'), ...]
```

Reply:
Good idea.  Will change in next pass.